src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsSlotsMercury.java

```diff
@@ -33,6 +35,7 @@ public void adsAllSlotsOnPage(String wikiName,
     MobileAdsBaseObject ads = new MobileAdsBaseObject(driver, testedPage);
     ads.verifyGptIframe(adUnit, MOBILE_TOP_LEADERBOARD, SRC);
     ads.verifyGptIframe(adUnit, MOBILE_IN_CONTENT, SRC);
+    ads.scrollToPosition(ARTICLE_FOOTER);
```

Comment:
Wouldn't it be better to scroll to the mobile prefooter instead of article footer?